.github/workflows/phpstan.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php-version }}
-          tools: phpstan, cs2pr
+          tools: phpstan:1.12, cs2pr
```

Comment:
@mdeweerd to stay on 1.12 before switching on v2

Reply:
Thanks for the head's up.